index.html

```diff
@@ -34,7 +35,8 @@ <h2 class="header__subtitle">Subtitle</h2>
 
       <section>
         <a class="link__question" href="#">START</a>
-      </section>
+      </section
```

Comment:
You removed the '>' put it back

Reply:
I have already closed the section tag with ">"